app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+# Create A New Web Application To Manage Nimbus Experiments
+
+* Status: proposed
+* Deciders: Kate Hudson, Jared Lockhart, TBD
+* Date: 2020-08-18
+
+## Context and Problem Statement
+
+Experimenter is a web application developed solely for the purpose of managing product experiments at Mozilla.  It was originally designed to manage the Shield experimentation workflow, and has expanded to include a comprehensive workflow for managing Normandy experiments.  Project Nimbus is being developed with very different core assumptions than the previous experimentation platforms, and it is unclear whether building on top of the existing Experimenter infrastructure or creating a new purpose specific application will best suit the project's needs.
+
+## Decision Drivers <!-- optional -->
+
+* Meeting Project Nimbus' needs in a rapid, continuous fashion with minimal downtime
+* Contributability by a large, distributed group of people with different backgrounds (web engineers, data engineers, data scientists)
+* Fitting cohesively into the FXA engineering pipeline/culture
+* Continuing to support the existing experimentation (Normandy) workflow
+
+## Considered Options
+
+* Build On Top Of Experimenter
+* Start A New Application In Python/Django
+* Start A New Application in TypeScript/Node
+
+## Decision Outcome
+
+Chosen option: TBD
+
+### Positive Consequences
+
+* [e.g., improvement of quality attribute satisfaction, follow-up decisions required, …]
+* …
+
+### Negative Consequences
+
+* [e.g., compromising quality attribute, follow-up decisions required, …]
+* …
+
+## Pros and Cons of the Options
+
+### Build On Top Of Experimenter
+
+Much of the existing Experimenter infrastructure can be reused, which gives Project Nimbus a well established framework to build on top of.
+
+- Pros
+  - Single source of truth for all past/future experiments
+    - Easily discover past results
+    - Correlate past/future experiments easily
+    - Generate high level reports for leadership/stakeholders across all experiment types
+    - Consistent APIs for dependent automated tasks (ie data engineering/science)
+  - Reuse existing components
+    - Database schemas are largely reusable
+    - Public APIs work seamlessly for Normandy and Nimbus experiments
+    - External service integrations (bugzilla/remote settings/email/auth0) work seamlessly
+  - Comprehensive development/testing/deployment infrastructure in place
+    - 100% unit testing coverage across Python/JS/TS
+    - Automated Selenium Integration tests for frontend
+    - Continuous Deployment
+- Cons
+  - Existing workflow/UI not applicable
+  - Some database schemas require refactoring to isolate Normandy and Nimbus concerns
```

Comment:
...however, this could be mitigated by creating new tables/models.

Reply:
Yeah that's what I meant, right now we're just naively using the same model for both, but it's very possible to split it up into pieces such that the nimbus focused code only interacts with db models that are specific to its concerns.  Using this approach it would be possible to have some shared db models for things that apply to both, allowing things like public apis and high level reporting to treat all experiments in a consistent fashion while isolating the Normandy and Nimbus specific models to their respective flows.